### ISSUE TITLE
can't change style format 

### ISSUE THREAD

**Issue by s4wet** [pvoznyuk/react-live-clock#17]
component page

```
import React  from 'react';
import Clock from 'react-live-clock';

export default class Mytime extends React.Component {
    render() {
        return(
        <Clock format={'HH:mm A'} ticking={true} timezone={'Asia/Bangkok'}  />
        )
    }
}
```
main page 

```
var des2 ={
  color: 'white',
  marginLeft: '10px'

}
<Mytime style={des2}  />

```

**Comment by pvoznyuk** [comment]
Hi @s4wet  
I am preparing a new release and I'll include your suggestion into it. 
It will be ready on weekend.

Thank you.

**Comment by pvoznyuk** [comment]
Hi @s4wet  it is fixed in the new version (3.0.0)
